### PR TITLE
Mint endpoint

### PR DESCRIPTION
**Pull request by altafan**
This adds the `/mint` endpoint for Liquid faucet.

After this is merged I'll publish the new image into the docker hub.

PS: The inner bitcoin and liquid faucet implementations have been merged, there was actually no reason to use an interface with 2 different implementations for calling the same commands (except for the liquid only `issueasset` rpc).

### CODE REVIEW

**Comment by tiero** [comment]
It should closes #23

File: config/main.go [hunk 1]
```diff
@@ -166,7 +166,7 @@ func NewTestConfig() Config {
 	c.electrs.port = "3002"
 
 	c.rpcServer.host = os.Getenv("ADDR")
-	c.rpcServer.port = "18443"
+	c.rpcServer.port = "18433"
```
Comment (tiero) [inline config/main.go:169]: Why it's changing?

**Comment by tiero** [comment]
Add example of `/mint` usage in Readme

File: config/main.go (hunk 1 above)
Comment (altafan) [reply config/main.go:169]: Nigiri's liquid node default port: https://github.com/vulpemventures/nigiri/blob/e02c2fdd0d93a8014df1f5dcf1027f97e88689fc/cli/constants/constants.go#L33

**Pull request merged**